piserver/app.py

```diff
@@ -22,15 +23,14 @@
 # INIT BLINDS SYSTEM RELATED COMPONENTS #
 temp_sensor = BME280TemperatureSensor() if app.config["USE_TEMP_SENSOR"] \
         else MockTemperatureSensor()
-smart_blinds_system =  SmartBlindsSystem( Blinds( None ), None, temp_sensor )
+#TODO: Init a motor hanndler here
 
-@app.route('/')
-def index():
-    return 'Server Works!'
-  
-@app.route('/greet')
-def say_hello():
-    return 'Hello from Server'
+# default empty schedule 
+app_schedule = BlindsSchedule( BlindMode.DARK, None, None )
```

Comment:
Why is the BlindSchedule initialized with the DARK mode specifically? I understand that a valid enum value has to be passed or else the app schedule fails, but could it lead to some weird behaviour if an arbitrary mode is added to an empty blind schedule?

I think it might be nice to have an INVALID or NONE mode just for initialization. But idk. What do you think?

Reply:
My thinking is that the system would periodically poll for whether or not there's a command (top priority), then check for a scheduled block, then finally the schedule defaults. I gave it DARK because I thought it would be a reasonable default behaviour. Without it, if the system is started for the first time and has nothing, what will it do? it'll poll and... then what? Idk, what do you think haha

Reply:
Oh i see, that's reasonable. If there is no command or anything in the schedule block then nothing happens anyways. I think you can leave it as is :).